lib/exosuit/instance.rb

```diff
@@ -44,5 +44,9 @@ def self.all
     def self.running
       all.select { |i| i.state.name == 'running' }
     end
+
+    def self.latest_ami
```

Comment:
I vote we instead call this `latest_ami_image_id` since the thing it returns is not an AMI but the image id of an AMI.